src/Login.js

```diff
@@ -0,0 +1,34 @@
+import { Button } from '@material-ui/core';
+import React from 'react';
+import './Login.css';
+import { auth, provider } from './firebase';
+import { actionTypes } from './reducer';
+import { useStateValue } from './StateProvider';
+
+function Login() {
+    const [{ }, dispatch] = useStateValue();
+    const signIn = () => {
+        auth
+            .signInWithPopup(provider)
+            .then((result) => {
+                dispatch({
+                    type: actionTypes.SET_USER,
+                    user: result.user,
+                })
+            })
+            .catch((error) => alert(error.message));
+    };
+    return (
+        <div className="login">
+            <div className="login_container">
+                <img src="https://upload.wikimedia.org/wikipedia/commons/6/6b/WhatsApp.svg" alt="" />
```

Comment:
change the logo to this logo i have made https://drive.google.com/file/d/1a4HNiEcdImH3EHqUf7tGEoOq4n7I2x4n/view?usp=sharing